cellx/tools/projection.py

```diff
@@ -32,13 +32,13 @@ class ManifoldProjection2D:
 
     Parameters
     ----------
-    images : list[str] or np.ndarray (N, W, H, C)
-        A list of image patches in np.ndarray format.
-    output_shape : tuple
-        Final size of individual image patches in the projection space.
+    images : list of str or (N, W, H, C) np.ndarray of int
```

Comment:
```suggestion
    images : list of str or (N, W, H, C) np.ndarray
```
This doesn't necessarily need to be an int. I suppose it could be anything in reality.